### PR TITLE
When setting directions, use walking mode & valid startpoint

### PR DESCRIPTION
**Pull request by DanielHeath**
Google maps doesn’t actually accept the string “Current Location”.

### CODE REVIEW

File: src/components/GoogleMapsLink.js [hunk 1]
```diff
@@ -12,19 +12,19 @@ class GoogleMapsLink extends React.Component {
...
+            from: "",
         },
     };
 
     googleMapsUrl(): string {
         var toAddr = this.props.to;
 
-        // FIXME: this.props.from
-        var start = "Current+Location";
+        var start = encodeURIComponent(this.props.from || "");
```
Comment (danni) [inline src/components/GoogleMapsLink.js]: Use `defaultProps` and add a comment explaining the meaning of ""

File: src/components/GoogleMapsLink.js [hunk 2]
```diff
@@ -12,19 +12,19 @@ class GoogleMapsLink extends React.Component {
...
         var query = encodeURIComponent(
             `${toAddr.streetAddressLine1()} ${toAddr.streetAddressLine2()}`
         );
 
-        return `https://maps.google.com/?saddr=${start}&addr=${query}`;
+        return `https://maps.google.com/?directionsmode=walking&saddr=${start}&daddr=${query}`;
```
Comment (danni) [inline src/components/GoogleMapsLink.js:32]: Can we do walking or tramming?

Comment (DanielHeath) [inline src/components/GoogleMapsLink.js:32]: You can only specify one of them; walking seemed like the safer option.

Comment (danni) [inline src/components/GoogleMapsLink.js:32]: Let's add a FIXME to do it based on whichever we're showing in the travel time widget once that works.

**Pull request merged**